src/core/components/checkbox/styles.ts

```diff
@@ -52,7 +52,10 @@ export const checkbox = ({ checkbox } = checkboxDefault) => css`
 		${focusHalo};
 	}
 
-	@supports (appearance: none) {
+	@supports (
+		(appearance: none) or (-webkit-appearance: none) or
+			(-moz-appearance: none)
+	) {
```

Comment:
is it worth pulling this out somewhere?

```suggestion
	@supports (${appearance}) {
```

Reply:
nbd tho

Reply:
Good shout, I expect this will come up again. The only place that makes sense right now is `@guardian/src-helpers` as it doesn't sound like a `@guardian/src-foundations` thing.